crates/compiler/builtins/bitcode/src/main.zig

```diff
@@ -145,6 +145,7 @@ comptime {
     exportStrFn(str.strTrimRight, "trim_right");
     exportStrFn(str.strCloneTo, "clone_to");
     exportStrFn(str.withCapacity, "with_capacity");
+    exportStrFn(str.strGraphemes, "str_graphemes");
```

Comment:
We don't normally put the `str_` prefix in front of the second argument here, as you can see from the others here. `exportStrFn` already prepends `str.` to the name, so this will be exported as `str.str_graphemes`, which is technically fine but not needed.

Reply:
I didn't want this to be named `graphemes` as we have a `graphemes.zig` with the same import name. I was unsure if there would be a namespace conflict, so prepended `str_` to the name, similar to `str_split`